.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Do we need to modify this to include `v9.0.0`?

Reply:
Not really. This configuration will execute the tests for all pull requests to any branch and also for all commits in the master branch (push means commits 😅 ):
```yml
on:
  push:
    branches: [ master ]
  pull_request:
```

Executing it for all PRs to any branch should be enough, right?

Reply:
Ah, got it!